kview/src/main/java/dev/ikm/komet/kview/mvvm/view/pattern/PropertiesController.java

```diff
@@ -210,6 +216,15 @@ private void initialize() {
         patternFieldsPane = patternFieldsJFXNode.node();
         patternFieldsController.setViewProperties(getViewProperties());
 
+        // +-----------------------------------
+        // ! Instances Toggle selected
+        // +-----------------------------------
+        Config instancesConfig = new Config(INSTANCES_URL)
+                .addNamedViewModel(new NamedVm("patternPropertiesViewModel", patternPropertiesViewModel));
```

Comment:
This isn't recommended as the parent controller really owns the `patternPropertiesViewModel`. The `patternPropertiesViewModel` variable name does not exist inside the `InstanceController`. I'm surprised it didn't throw an runtime exception. 
A quick fix would be:
remove the line: 223. or call 👍🏼 
```java
  .updateViewModel("simpleViewModel",  (viewModel) -> { 
      // viewModel.addProperty(property, value);
  } );
```

When injecting view models the following can are allowed:
- A variable name must exist in controller code via `@InjectViewModel`. a NamedVm is a key value of the variable name and view model respectively.
- Caller can construct their own instance to be injected, but the variable name must exist. FXMLMvvm.make() will  use caller's and not instanciate a new one.

Reply:
This is not the InstancesController, it's the PropertiesController class.

Reply:
Maybe i'm missing what you are saying but below shows line 222 inside of `PropertiesController`:
```java
Config instancesConfig = new Config(INSTANCES_URL)
                .addNamedViewModel(new NamedVm("patternPropertiesViewModel", patternPropertiesViewModel));
```
Is the above code stating `new Config(INSTANCES_URL)`? That is the fxml and controller being created via `FXMLMvvm.make()`. Here it is the code to create the FXML and InstancesController i thought (child window of PropertiesController).

Reply:
And this would match what is inside the InstancesController

`Config instancesConfig = new Config(INSTANCES_URL)
                .addNamedViewModel(new NamedVm("patternViewModel", patternViewModel));`